csrankings.csv

```diff
@@ -14310,6 +14313,7 @@ Sheng Li 0001,University of Georgia,http://www.cs.uga.edu/~shengli,DEncVcYAAAAJ
 Sheng Li 0002,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6082,NOSCHOLARPAGE
 Sheng Li 0003,Harbin Institute of Technology,http://ir.hit.edu.cn/~ls,NOSCHOLARPAGE
 Sheng Wei 0001,Rutgers University,http://eceweb1.rutgers.edu/~sw891,9fBhPy0AAAAJ
+Sheng Zhang 0001,Nanjing University,http://cs.nju.edu.cn/sheng/,2fqdkfMAAAAJ

```

Comment:
"Assistant researcher": this title does not indicate that this person is eligible ("professor" and similar yes, this, not obviously so).

Reply:
Thanks, I will recheck it and update it again.